include/deal.II/lac/lapack_full_matrix.h

```diff
@@ -478,14 +484,34 @@ class LAPACKFullMatrix : public TransposeTable<number>
    */
   void invert ();
 
+  /**
+   * Solve the linear system with right hand side. The matrix should
+   * be either triangular or LU factorization should be previously computed.
+   *
+   * The flag transposed indicates whether the solution of the transposed
+   * system is to be performed.
+   */
+  void solve(Vector<number> &v,
+             const bool transposed) const;
```

Comment:
maybe give it a default of `false`? That's the usual case.

Also, can you state that the solution is put back into the vector w?